lib/semanage.c

```diff
@@ -297,6 +297,7 @@ int set_seuser (const char *login_name, const char *seuser_name)
 done:
 	semanage_seuser_key_free (key);
 	semanage_handle_destroy (handle);
+	reset_selinux_handle();
```

Comment:
The call is probably only needed after a successful `semanage_commit()`, so it might also go with the `r = 0` statement.

Reply:
That makes sense, so I've moved the statement